guides/hhvm/03-advanced-usage/02-repo-authoritative.md

```diff
@@ -57,6 +47,17 @@ src/b.php
 lib/c.php
 ```
 
+### Specific directories
+
+```
+hhvm --hphp -t hhbc --module src --module vendor --ffile public/index.php
+```
+
+Flag | Description
+-----|------------
+`--module` | specifies a directory (module) to include in the repository
+`--ffile` | forces a specific file to include in the repository
```

Comment:
Being _in the repository_ is a rather confusing way to describe it. Since `--input-dir` files are not in the repository, but can be executed.